source/extension-manifest.json

```diff
@@ -207,14 +207,14 @@
                         "resultSelector": "jsonpath:$.[*]"
                     },
                     {
-                        "name": "OctopusAllProjects",
-                        "endpointUrl": "$(endpoint.url)/api/projects/all",
-                        "resultSelector": "jsonpath:$.[*]"
+                        "name": "OctopusListProjects",
+                        "endpointUrl": "$(endpoint.url)/api/projects?partialName=$(PartialNameFilter)",
+                        "resultSelector": "jsonpath:$.Items[*]"
```

Comment:
When using our list endpoint / index responders, we have to squirrel down into the `Items` from the response.